docs/Developers/Executing_Tests.rst

```diff
@@ -56,8 +56,12 @@ To run our tests, you can also use ``python3 setup.py test``
 
 .. note::
 
-    You will not get a test coverage of 100% - the coverage on the
-    website is merged for several python versions.
+    You will not get a test coverage of 100% locally! - the coverage is
+    measured across builds against several python versions. The coverage
```

Comment:
I think this line is redundant and isn't making things clear enough. What do you think about removing it for the sake of the next line?